src/amber/environment/settings.cr

```diff
@@ -13,13 +21,21 @@ module Amber::Environment
     property process_count : Int32
     property redis_url
     property secret_key_base : String
-    setter session : Hash(String, Int32 | String)
     property secrets : Hash(String, String)
     property ssl_key_file : String
     property ssl_cert_file : String
+    property logger : Logger?
 
     YAML.mapping(
-      colorize_logging: {type: Bool, default: true},
+      logging: {type: LoggingType, default: {
+        severity: "info",
+        color:    true,
+        time:     false,
+        level:    false,
```

Comment:
Why is level a boolean?

Reply:
this is to enable printing the output string `(Debug)`, `(INFO)`. Severity if to assign the type.

Reply:
Semantically, `level` does seem strange for this. If I understand correctly, this is used to determine whether or not the `severity` is output by the logger, correct?

It may be a little long, but `output_severity` might work better.

What is common in other frameworks using `severity` vs `level` for logging?

Reply:
We can prefix the booloan with `show_` which dictates some intention